src/cmd/flux-jobs.py

```diff
@@ -170,26 +180,9 @@ def fetch_jobs_flux(args):
 
     states = 0
     for state in args.states.split(","):
-        # Note
-        # pending = depend & sched
-        # running = run & cleanup
-        if state.lower() == "depend":
-            states |= flux.constants.FLUX_JOB_DEPEND
-        elif state.lower() == "sched":
-            states |= flux.constants.FLUX_JOB_SCHED
-        elif state.lower() == "pending":
-            states |= flux.constants.FLUX_JOB_PENDING
-        elif state.lower() == "run":
-            states |= flux.constants.FLUX_JOB_RUN
-        elif state.lower() == "cleanup":
-            states |= flux.constants.FLUX_JOB_CLEANUP
-        elif state.lower() == "running":
-            states |= flux.constants.FLUX_JOB_RUNNING
-        elif state.lower() == "inactive":
-            states |= flux.constants.FLUX_JOB_INACTIVE
-        elif state.lower() == "active":
-            states |= flux.constants.FLUX_JOB_ACTIVE
-        else:
+        try:
+            states |= state_const_dict[state.lower()]
+        except KeyError:
             print("Invalid state specified", file=sys.stderr)
```

Comment:
Suggestion: print the name of the invalid state in the error message.

Reply:
good idea, and I should add a test for an invalid state